src/components/ebay-notice/index.marko

```diff
@@ -26,22 +27,24 @@ $ var isLightGuidance = isSection && !input.status;
         input.fillWindow && `${type}-notice--fill`,
         input.class
     ]>
-    <if(!isLightGuidance)>
+    <if(!isLightGuidance && (isIconVisible || input.title))>
```

Comment:
Why is this extra check needed for `isIconVisible` or `input.title`? I see below that there is already a check for `isIconVisible`. 
The question is, is this needed? Is there a better way to simplify it since it is a bit complex?

Reply:
If there is no icon and no title, there is no need to render this whole block, because it will be empty. (and it will create extra space)

Reply:
Will try to simplify it a bit for more clarity